backends-velox/src/test/scala/org/apache/gluten/execution/TestOperator.scala

```diff
@@ -2078,4 +2078,12 @@ class TestOperator extends VeloxWholeStageTransformerSuite with AdaptiveSparkPla
       checkGlutenOperatorMatch[SortExecTransformer]
     }
   }
+
+  // Open it after fix
```

Comment:
Better change to `// Enable the test after fixing https://github.com/apache/incubator-gluten/issues/6827`, thanks.